src/components/guide/index.js

```diff
@@ -0,0 +1,190 @@
+      if(competencyName === comp.name) {
+        this.setState({displayedCompetency: comp});
+      }
+    });
+  }
+  expandedIntro(text) {
+    if(this.state.expandedIntro) {
+      return (
+        <p>{text}</p>
+      );
+    }
+  }
+  handleReadMore() {
+    this.setState((prevState) => ({expandedIntro: !prevState.expandedIntro}));
+  }
+  introduction() {
+    const {introduction} = this.state.displayedCompetency;
+
+    let intro = introduction.split(".", 1)[0];
+    intro = `${intro}.`;
+    const readMore = introduction.replace(intro, "").trim();
+    return {intro, readMore};
+  }
+  selectBoxOrTabs() {
+    const {displayedCompetency} = this.state;
+
+    if(smallScreen()) {
+      return (
+        <div>
+          <select
+            value={displayedCompetency.name}
+            className={style.mobileSelect}
+            onChange={(e) => this.displayCompetency(e)}
+          >
+            {this.state.competencies.map((competency) => (
+              <option value={competency.name}>{competency.name}</option>
+            ))
+            }
+          </select>
+          <p className={style.mobileBadge}>
+            <img src={`https://cdn.traitify.com/assets/images/js/${this.tabImage(displayedCompetency.name)}`} alt={`${displayedCompetency.name} badge`} />
+          </p>
+        </div>
+      );
+    } else {
+      return (
+        <ul className={style.tabs}>
+          {this.state.competencies.map((competency, index) => (
+            <li
+              className={competency.name === displayedCompetency.name ? style.tabActive : null}
+              key={competency.id}
+            >
+              <a
+                href={`#tab-${index}`}
+                tabIndex={0}
+                onKeyPress={() => this.displayCompetency(competency.name)}
+                onClick={() => this.displayCompetency(competency.name)}
+                name={competency.name}
+              >
+                <img src={`https://cdn.traitify.com/assets/images/js/${this.tabImage(competency.name)}`} alt={`${competency.name} badge`} />
+                <br />
+                {competency.name}
+              </a>
+            </li>
+          ))}
+        </ul>
+      );
+    }
+  }
+  setGuide() {
+    const params = {assessmentId: this.props.assessmentID};
+    this.props.traitify.graphqlQuery("/interview_guides/graphql", guideQuery({params}))
+      .then((response) => {
+        if(response.errors) { this.setState({errors: response.errors}); return; }
+        const {competencies} = response.data.guide;
+        this.setState({competencies, displayedCompetency: competencies[0]});
+      });
+  }
+  stringToListItems(entity) {
+    let entities = entity.replace("\n", "<br/>").split("<br/>");
+    entities = entities.map((e) => <li>{e}</li>);
+
+    return (
+      <ul>
+        {entities}
+      </ul>
+    );
+  }
+  tabImage(tab) {
+    switch(tab) {
+      case "Solving Problems": return "openness.png";
```

Comment:
When testing pulling them from the assessment it would work with English because the type name that im switching on here is correct, but when the locale is updated and the assessment is re-fetched in another language it errors out bc of a name like "Openheild". My thought is it is best to leave this as I had it before and since it wasn't clear how else to match this data

Reply:
I left the updated code where it pulls the badge from the assessment in this pr for now if you want to take a look at it and see if there's a better way

Reply:
Dang, good point. I think you were right the first time then. Mind switching it back?